### DISCUSSION TITLE
PostgreSQL

### DISCUSSION THREAD

**Discussion by KenDarlingPAF** [JanDeDobbeleer/oh-my-posh#1206] [Troubleshoot]
Hello everyone, has anyone done anything with PostgreSQL?  I would like to posh it up, but have not found much about it

**Comment by JanDeDobbeleer** [comment]
What would you expect in this case? If you can give an example of the information you'd like to see we can check what we can do.

**Reply by KenDarlingPAF** [reply]
When I am logged in to my PostgreSQL shows the current database that I am connected in as _mytestdb=#_ would like to do something with that, to get started, this all may just be that I need to learn how to customize themes better.

**Reply by JanDeDobbeleer** [reply]
Do you have any environment variables that contain this information?

**Reply by KenDarlingPAF** [reply]
it looks like it does, I can then try using the environment variables segment?

**Reply by JanDeDobbeleer** [reply]
Yes, or even text with a template (same thing)

**Reply by KenDarlingPAF** [reply]
![image](https://user-images.githubusercontent.com/47124065/141192078-414b2f9a-0aa8-4c5f-829a-fdf88e8d6049.png)

I am trying to do something with the mytestdb=#, looking at the Environment Variables on the PostgreSQL documentation looks like it should be PGDATABASE 

so looking at the sample config for text I am not sure where or how to declare the .Env.VarName

{
  "type": "text",
  "style": "plain",
  "foreground": "#E06C75",
  "properties": {
    "prefix": "",
    "text": " \u276F"
  }
}

Sorry, I feel like I being a pain

**Reply by JanDeDobbeleer** [reply]
You should be able to set `"text": "{{ .Env. PGDATABASE }}"`
